scripts/firecrawl_sdk_research.py

```
@@ -43,45 +48,220 @@ def get_api_key() -> str:
+            
+            result_dict = {
+                "url": url,
+                "title": title,
+                "description": description,
+                "markdown": getattr(r, "markdown", ""),
+            }
+            if metadata:
+                # Convert metadata to dict if possible
+                try:
+                    result_dict["metadata"] = metadata.model_dump() if hasattr(metadata, "model_dump") else {}
+                except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)